browser/app/js/objects/Path.js

```diff
@@ -34,7 +34,12 @@ export const Path = ({ currentBucket, currentPrefix, selectPrefix }) => {
         dirPath.push(dir)
         let dirPath_ = dirPath.join("/") + "/"
         return (
-          <div className="path__item">
+          <div
+            className={classNames({
+              path__item: true,
+              "path__item--active": currentPrefix === dirPath_
+            })}
```

Comment:
We would need to move `key={i}` on line 43 to here as `div` is now the enclosing tag for what we are returning. Currently, we get the following warning:
![screen shot 2018-09-27 at 11 20 54 am](https://user-images.githubusercontent.com/27314853/46166756-cb5b6b00-c248-11e8-85cd-2feee1397801.png)